documentation/src/sops/light_measurement_devices/goprohero10.md

```diff
@@ -62,14 +52,11 @@ In our project, we strive to measure consecutively in intervalls without having
 
 
 Export/Access Data
-    1. Open the slide on the side of the GoPro Hero 10 and take out the microSD card
-    2. If your storage device does not have a slot for a microSD card, use a SD-to-microSD-adapter to connect the microSD carf to your device
-    3. Export the preferred data. The videos are saved as .mp4-files and photos as .jpg-files. Besides these, .THM-, .GPR- and .LVM-files are also exportable
-    3.1. .THM-files: Scaled down image format for thumbnails (160x120 pixels)
-    3.2. .GPR-files: Compressed image-file that contains more data, allowing for improved picture quality
-    3.3..LRV-file: Low-quality video file, that is used for showing previews on the mobile App "Quik".
-    4. Save and rename the data as you wish
-    5. Alternatively: Delete copied data to free up storage on the microSD-card
+1. Open the slide on the side of the GoPro Hero 10 and take out the microSD card
+2. If your storage device does not have a slot for a microSD card, use a SD-to-microSD-adapter to connect the microSD carf to your device
+3. Export the preferred data. The videos are saved as .mp4-files and photos as .jpg-files. Besides these, .THM-, .GPR- and .LVM-files are also exportable. First, THM-files are a scaled down image format for thumbnails (160x120 pixels). Second, GPR-files are compressed image-files that contain more data, allowing for improved picture quality. Third, an LRV-file is a low-quality video file, that is used for showing previews on the mobile App "Quik".
```

Comment:
Thanks for fixing the numbering. I think that this is too much text though. 
You _can_ have hierarchical numbering, it's just not done by 3.1 > 3.1.1 but instead done with indent. 
So, basically:

```
1. level 1
2. level 1
3. level 1
indent1. level 2
indent2. level 2
```

markdown will even figure out the number itself, when you write it like this:
```
1. level 1
indent1. level 2
indent1. level 2
1. level 1
```

It will still look like this:

1. level 1
<t> 1. level 2
<t> 2. level 2
1. level 1

I am requesting changes before accepting (also to practise this function on GitHub 😄 )